src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = sourceString
+    .split(';')
+    .filter((style) => style.trim().length !== 0)
+    .reduce((acc, current) => {
+      const [key, value] = current.split(':');
+
+      acc[key.trim()] = value.trim();
+
+      return acc;
```

Comment:
Everything is great, but chaining contributes to the worse readability of the code. Try splitting the code into constants.